CONTRIBUTING.md

```diff
@@ -189,9 +189,15 @@ in the body of the pull request to indicate tasks are pending.
 
 At this stage, one should expect comments and review from other contributors. You
 can add more commits to your pull request by committing them locally and pushing
-to your fork until you have satisfied all feedback.
+to your fork.
 
-Note: Code review is a burdensome but important part of the development process, and as such, certain types of pull requests are rejected. In general, if the **improvements** do not warrant the **review effort** required, the PR has a high chance of being rejected. It is up to the PR author to convince the reviewers that the changes warrant the review effort, and if reviewers are "Concept NACK'ing" the PR, the author may need to present arguments and/or do research backing their suggested changes.
+As the pull request author you are expected to address any review comments
```

Comment:
"As the pull request author you are expected to address any review comments
before your pull request is merged" - What do you mean by 'address'?

Reply:
If @MarcoFalke responds to this comment and explains the meaning of 'address' in this context, it will be 'addressing' review comments of this pull request.

In other pull requests it can be related to code and sometimes reviewers may not agree with certain changes or rationale provided for the code.

Reply:
Make sense. I thought it could sound 'implement' what it is not true since we may not agree with all feedbacks. But it sounds like 'reply' or 'consider' what makes sense. Thank you for explaining, just a doubt related to this meaning.

Reply:
Thanks, changed a bit.